src/webgazer.js

```diff
@@ -395,11 +395,10 @@
         videoElement.style.display = 'none';
 
         // set the video source as the stream
-        if ("srcObject" in videoElement) {
+        if ("srcObject" in videoElement){
```

Comment:
Space between closed bracket and open brace shouldn't be removed.

(Not sure how picky we want to be with our reviews so feel free to ignore :) )